Tensile/Tests/extended/direct_to_lds/dtl_sgemm.yaml

```diff
@@ -1,5 +1,5 @@
 TestParameters:
-  marks: [skip-gfx900, skip-gfx906, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030] # not supported by arch
+  marks: [skip-gfx900, skip-gfx906, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030, skip-gfx1100, skip-gfx1101, skip-gfx1102] # not supported by arch
```

Comment:
This also includes change from other commit (#1557).
It is probably better to add your change on top of 5.3 release code based on your code change log.

Reply:
Extra skips in this test case shouldn't break anything

Reply:
> Extra skips in this test case shouldn't break anything

I understand that. But, I think it is better to make your change as simple as possible.

Reply:
Updated